bcda/web/api.go

```diff
@@ -33,31 +33,6 @@ import (
 
 var qc *que.Client
 
-/*
-  	swagger:route GET /api/v1/ExplanationOfBenefit/$export bulkData bulkEOBRequest
-
-	Start explanation of benefit data export
-
-	Initiates a job to collect explanation of benefit data from the Blue Button API for your ACO.
-
-	Produces:
-	- application/fhir+json
-
-	Security:
-		bearer_token:
-
-	Responses:
-		202: BulkRequestResponse
-		400: badRequestResponse
-		401: invalidCredentials
-		429: tooManyRequestsResponse
-		500: errorResponse
-*/
-
-func bulkEOBRequest(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think we are going to want to keep `/ExplanationOfBenefit` and `/Coverage` around for a little while, for backwards compatibility.